exporter/metrics.go

```diff
@@ -209,7 +209,9 @@ func asFloat64(value interface{}) (*float64, error) {
 		f = v
 	case primitive.DateTime:
 		f = float64(v)
-	case primitive.A, primitive.ObjectID, primitive.Timestamp, primitive.Binary, string, []uint8, time.Time:
+	case primitive.Timestamp:
+		f = float64(v.T)
```

Comment:
Can we use `time.Unix()` or `time.UnixMilli()` or any of those functions to make it more clear?

Reply:
I guess you could do `f = float64(time.Unix(v.T, 0).Unix())` but I'm not sure that makes it clearer!